game.html

```diff
@@ -3,21 +3,37 @@
 (function()
 {
 var answer = 505;
-
+var withinOneHundred = 'Getting closer, You\'re within 100! Keep guessing!';
+var guessAgain = 'Muerto, you get an Eybeball, guess again';
 var guess = prompt ('How many Gumballz in the Gumball machine?');
+var tooMany = 'That is too many tries, you dont get a gumball!'
+var correct = ('Correcto, you get a Gumball!')
 guess = Number.parseInt(guess);
+var numTries = 0;
+
 while (true) {
 if (guess === answer){
-alert ('Correcto, you get a Gumball!');
+alert (correct);
 break;
-} else if (guess < answer) {
-  guess = Number.parseInt(prompt('Muerto, you get an Eyeball, guess again'));
+}
+
```

Comment:
The while and if blocks were hard to follow because of lack of indenting. See me (or another TA) if you want pointers.